tests/integration/twonodes.nim

```diff
@@ -11,43 +11,66 @@ export codexclient
 export nodes
 
 template twonodessuite*(name: string, debug1, debug2: bool, body) =
+  twonodessuite(name, $debug1, $debug2, body)
+
+template twonodessuite*(name: string, debug1: string, debug2: bool, body) =
+  twonodessuite(name, debug1, $debug2, body)
+
+template twonodessuite*(name: string, debug1: bool, debug2: string, body) =
+  twonodessuite(name, $debug1, debug2, body)
```

Comment:
Most probably there is better way how to do this, so please if you have some idea please shoot ;-)

Reply:
Oh, I was not sure if you can do such a thing in Nim, great thanks!